src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -1101,5 +1101,478 @@
     <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true"/>
     <field-descriptor name="noteText" column="ACH_NTE_TXT" jdbc-type="VARCHAR"/>
   </class-descriptor>
+  
+  <class-descriptor
+	class="edu.cornell.kfs.fp.document.CorporateBilledCorporatePaidDocument"
+	table="CU_FP_CBCP_DOC_T">
+	<field-descriptor name="documentNumber" column="FDOC_NBR"
+		jdbc-type="VARCHAR" primarykey="true" index="true" />
+	<field-descriptor name="versionNumber" column="VER_NBR"
+		jdbc-type="BIGINT" locking="true" />
+	<field-descriptor name="objectId" column="OBJ_ID"
+		jdbc-type="VARCHAR" index="true" />
+	<field-descriptor name="nextTargetLineNumber" column="FDOC_NXT_LINE_NBR"
+		jdbc-type="INTEGER" />
+	<field-descriptor name="autoApprovedIndicator" column="AUTO_APPROVED_IND"
+		jdbc-type="VARCHAR"
+		conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion" />
+
+	<reference-descriptor name="documentHeader"
+		class-ref="org.kuali.kfs.sys.businessobject.FinancialSystemDocumentHeader"
+		auto-retrieve="true" auto-update="object" auto-delete="object" proxy="true">
+		<foreignkey field-ref="documentNumber" />
```

Comment:
If you're splitting up the opening tags along multiple lines, I would suggest using a different level of indentation than that of the sub-elements. This would make the XML easier to read, and would be more consistent with how we indent our code.